submissions/description/broken_authentication_and_session_management/privilege_escalation/template.md

```diff
@@ -2,32 +2,37 @@
+Authentication and session management controls can be bypassed through a variety of ways including, calling an internal post authentication page, modifying the given URL parameters, by manipulating the form, or by counterfeiting sessions. The authentication method for this application can be bypassed by an attacker which enables them to access a privileged user’s account and functionality, giving them access to more resources or functionality within the application. This could include viewing or editing sensitive customer data, and viewing or editing other user permissions.
 
-A malicious attacker can perform a variety of actions through bypassing authentication methods and is limited only by the permissions of the user account that they have access to. This could include viewing or editing sensitive customer data, viewing or editing other user permissions, and taking over user accounts. This privilege escalation by the malicious attacker enables them to have access to more resources or functionality within the application.
+## Business Impact
 
-Vulnerability Specifics to the Application:
+The impact of privilege escalation through broken authentication controls can vary in severity depending on the degree of access to resources or functionality the malicious attacker is able to gain. An attacker with the ability to access, delete, or modify data from within the application could result in reputational damage for the business through the impact to customers’ trust. This can also result in indirect financial cost to the business through fines and regulatory bodies if sensitive data is accessed. The severity of the impact to the business is dependent on the sensitivity of the data being stored in, and transmitted by the application.
 
-The authentication method for {{application}} at {{url}} is being bypassed through {{action}}. This allows a malicious attacker to perform {{action}} which abuses the permissions of the privileged user.
+## Steps to Reproduce
 
-## Business Impact
+1. Enable a HTTP interception proxy, such as Burp Suite or OWASP ZAP
+1. Use a browser to navigate to: {{URL}}
+1. Login to User Account A
+1. Using the HTTP interception proxy, forward the following request to the endpoint:
 
-The impact of privilege escalation can vary in severity depending on the degree of access to resources or functionality the malicious attacker is able to gain. Generally, privilege escalation can result in the modification or theft of data, including any Personally Identifiable Information (PII) stored within the application. This can lead to financial loss from a compromised application, legal ramifications, and reputational damage of {{customer-name}} and their users.
+```HTTP Request
```

Comment:
This should probably just be `HTTP`: as per: https://github.com/github/linguist/blob/c497388e7f88736d7a2fe2735f64366b41a4b5f8/lib/linguist/languages.yml#L2502

Reply:
Cheers, please see latest commit @vortexau